charts/app-config/values-integration.yaml

```
@@ -60,6 +60,8 @@ _alb-ingress-group-backend: &alb-ingress-group-backend
 emergency-banner-redis:
   - &emergency-banner-redis redis://whitehall-admin-redis/1
 
+enableSearchAutocomplete: "true"
```

```suggestion
enableSearchAutocomplete: true
```

As per suggestion above about using this as a boolean rather than the contents of the env var